src/components/FwbInput/FwbInput.vue

```diff
@@ -60,6 +61,7 @@ interface InputProps {
   required?: boolean
   size?: InputSize
   type?: InputType
+  autocomplete?: AutoFill
```

Comment:
will we loose a lot if we set it as `'on' | 'off' | string` for now?

Reply:
Ok. Because TypeScript fails to handle `AutoFill`, I think we can compromise.